src/main/java/com/fvv/bookstore/dao/EmployeeDAOImpl.java

```diff
@@ -146,4 +147,65 @@ public Employee findEmployee(final Long id) throws PersonNotFoundException, DaoE
 		} 
 		return employee;
 	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public List<Employee> listEmployeesByName(String name) throws PersonNotFoundException, DaoException {
```

Comment:
there are duplicate code between this method and the listEmployees method, please unified both like you already did in other methods b4

Reply:
ok